.github/workflows/druid-test-image.yml

```diff
@@ -0,0 +1,36 @@
+name: Build Druid image for tests
+
+on:
+  pull_request:
+    paths:
+      - "scripts/test-druid-micro.dockerfile"
+      - ".github/workflows/druid-test-image.yml"
+
+jobs:
+  release:
+    name: 
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Authenticate GCloud
+        uses: google-github-actions/auth@v1
+        with:
+          credentials_json: "${{ secrets.RILL_BINARY_SA }}"
+
+      - name: Set up GCloud SDK
+        uses: google-github-actions/setup-gcloud@v1
+
+      - name: Build & Publish Rill docker image
+        run: |-
+          cd scripts
+
+          gcloud auth configure-docker
+
+          docker build -f test-druid-micro.dockerfile --build-arg DRUID_VERSION=${{ env.DRUID_VERSION }} -t gcr.io/rilldata/test-druid-micro:${{ env.DRUID_VERSION }} .
+          docker tag gcr.io/rilldata/test-druid-micro:${{ env.DRUID_VERSION }} gcr.io/rilldata/test-druid-micro
+          docker push gcr.io/rilldata/test-druid-micro:${{ env.DRUID_VERSION }}
+          docker push gcr.io/rilldata/test-druid-micro
```

Comment:
What about `druid-micro`, seems like that's what the image contains? Having `test` in the name sounds like the image runs tests, but it doesn't.

If you want `test` in the name, can we have it after the `druid` prefix (like `druid-micro-test`)? Having domain-scoped prefixes makes things easier to navigate/discover in listings

Reply:
`test` describes the purpose actually (ie it doesn't affect production functionality).